src/components/order-service/order-service.controller.ts

```diff
@@ -418,7 +418,11 @@ export class OrderServiceController implements angular.IController {
       var readyCondition: any = _.find(conditions, {type: "Ready"});
 
       this.ctrl.orderComplete = readyCondition && readyCondition.status === 'True';
-      this.ctrl.error = _.find(conditions, {type: "ProvisionFailed"});
+      this.ctrl.error = _.find(conditions, function(condition: any) {
+        return condition.reason === "ProvisionFailed" ||
+               condition.reason === "ProvisionCallFailed" ||
+               condition.reason === 'ErrorCallingProvision';
+      });
```

Comment:
I am getting the type going to 'Failed' now that I have updated to the latest origin-service-catalog.